packages/types/src/index.ts

```diff
@@ -365,6 +365,7 @@ export interface DriverHelpers {
   isPackageOrBundle: (app: string) => boolean;
   duplicateKeys: <T>(input: T, firstKey: string, secondKey: string) => T;
   parseCapsArray: (cap: string | string[]) => string[];
+  generateDriverLogPrefix: (obj: Object, sessionId?: string) => string;
```

Comment:
This is weird, and I had to look it up.

`Object` allows primitives, but not `null` nor `undefined`.  While that may actually work, it's not the intent of this function.  You probably mean `object`, which means an "actual" object.  

And again, while that might actually work, the only thing we really expect `obj` to be is an object implementing `Core`.  So I think `Object` should be `Core`.

I'm also not sure why some things go in "driver helpers" and not others.  But that's neither here nor there.

> However--and I view this as a major shortcoming of TypeScript--neither `object` nor `Object` understand if the object in question extends `Object.prototype`.  This means, given `const obj: object = Object.create(null)` _or_ `const obj: Object = Object.create(null)`  and then `generateDriverLogPrefix(obj)`, TS will _not_ complain that the `constructor` property does not exist on `obj` (it doesn't), and you will get a runtime error.

Reply:
changed type to Core